packages/hint/docs/user-guide/api/using-api.md

```diff
@@ -14,6 +14,15 @@ To use the API, the first thing you need to do is import the class `Analyzer`.
 import { Analyzer } from 'hint';
 ```
 
+This example is using TypeScript, so you'll also need to import some type definitions to
+enable Intellisense support and a successful TypeScript build. Any types you need are included with Webhint and are
+located in the `hint/dist/src/lib/types` directory. Two types we'll be using in the example are
```

Comment:
Hi @RyannosaurusRex 👋

Nice catch! One question I have is about the import path. We are [exporting the types directly on hint](https://github.com/webhintio/hint/blob/314bc896711b361f6cdf28f5b95ff4334fb99a92/packages/hint/src/lib/index.ts#L3) so people shouldn't have to access `hint/dist` (`dist` should be considered private). Can you see if that works in your code and update the message accordingly?

Thanks!